src/drivers/code_hosting/activate.sh

```diff
@@ -14,6 +14,10 @@ function activate_driver_for_code_hosting {
     activate_driver 'code_hosting' 'github'
   elif [ "$origin_hostname" == 'bitbucket.org' ]; then
     activate_driver 'code_hosting' 'bitbucket'
+  elif [ "$(contains_string "$origin_hostname" github)" == true ]; then
```

Comment:
I don't like this as it. Its not a bad assumption but feels wrong to me to do. Ideally I'd like us to just read the ssh identities file to get the actual hostname that is associated with the custom host. 

Thats definitely harder and more work so I'm okay with this as is, if we document it somewhere that if you are using ssh identities and want it use github / bitbucket, ensure the custom hostname includes that.